samcli/lib/build/app_builder.py

```diff
@@ -120,11 +122,11 @@ def build(self):
             LOG.info("Building layer '%s'", layer.name)
             if layer.build_method is None:
                 raise MissingBuildMethodException(
-                    f"Layer {layer.name} cannot be build without BuildMethod. Please provide BuildMethod in Metadata.")
-            result[layer.name] = self._build_layer(layer.name,
-                                                   layer.codeuri,
-                                                   layer.build_method,
-                                                   layer.compatible_runtimes)
+                    f"Layer {layer.name} cannot be build without BuildMethod. Please provide BuildMethod in Metadata."
+                )
+            result[layer.name] = self._build_layer(
+                layer.name, layer.codeuri, layer.build_method, layer.compatible_runtimes
```

Comment:
We need to using the same version of black. I have seen lots of formatting back and forth.

Reply:
I am using `black, version 19.10b0` which is specified here https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md
What version do you have?

Reply:
```
(samcli37) ~/P/aws-sam-cli ❯❯❯ black --version                                                                                                                               ✘ 2
black, version 19.10b0
(samcli37) ~/P/aws-sam-cli ❯❯❯ make black
black samcli/* tests/*
All done! ✨ 🍰 ✨
379 files left unchanged.
(samcli37) ~/P/aws-sam-cli ❯❯❯
```